examples/gno.land/r/gnoland/blog/admin.gno

```diff
@@ -5,8 +5,8 @@ import (
 	"strings"
 
 	"gno.land/p/demo/avl"
-	"gno.land/p/demo/dao"
-	"gno.land/r/gov/dao/bridge"
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/gov/dao"
```

Comment:
If I'm not mistaken, this import combines two elements:  
1. The V1 implementation of gov/dao, which will not be used when we transition to V2.  
2. The long-term type for proposals (i.e., dao.ProposalRequest), which we expect to be compatible with V2, V3, and so on.  


If this is correct, I believe we should separate the r/gov/dao implementation (r/gov/dao/v1) from the generic types, possibly just ProposalRequest, and place them in r/gov/types.

Reply:
I was mistaken; gov/impl/v3 is the implementation, and this package focuses more on the types. Let me re-read it, and for now, please ignore my comment (but do not resolve it yet).

Reply:
gno.land/r/gov/dao is the proxy, I changed the package name after your request. Actual and future realms must only import that realm to create and query proposals. The actual v3 implementation is living on gno.land/r/gov/dao/v3/impl